stable/kong/templates/deployment.yaml

```diff
@@ -23,6 +23,12 @@ spec:
         app: {{ template "kong.name" . }}
         release: {{ .Release.Name }}
     spec:
+      {{- if .Values.image.pullSecrets }}
```

Comment:
You would also need to update migration job.

Reply:
thanks for the quick turnaround. Updated the migrations template with `imagePullSecrets`.